src/common/Connection.ts

```diff
@@ -263,6 +263,9 @@ export class CDPSession extends EventEmitter {
     this._sessionId = sessionId;
   }
 
+  /**
+   * @internal
+   */
```

Comment:
I don't think this is internal, as it's documented here: https://pptr.dev/#?product=Puppeteer&version=v10.1.0&show=api-cdpsessionconnection

Could we update this PR to add documentation for this method, please?

Reply:
Sure, I will mark it as public